src/main/java/com/google/devtools/build/lib/actions/ActionExecutionContext.java

```diff
@@ -455,7 +455,7 @@ public <T extends ActionContext> T getContext(Class<T> type) {
    * Report a subcommand event to this Executor's Reporter and, if action
    * logging is enabled, post it on its EventBus.
    */
-  public void maybeReportSubcommand(Spawn spawn) {
+  public void maybeReportSubcommand(Spawn spawn, String spawnRunner) {
```

Comment:
Mark `spawnRunner` as `@Nullable` (since the method it passes to is), or add an explicit non-null check (if that is important semantically).